components/content-search/SearchItem.js

```diff
@@ -87,6 +100,7 @@ SearchItem.defaultProps = {
 	id: '',
 	searchTerm: '',
 	isSelected: false,
+	renderType: null,
```

Comment:
@rmorse thoughts on putting the logic you added on L53 here as a sensible default? That way the logic from L52:L57 can be simplified as `renderType` would always have a function (unless the prop has a value type it shouldn't but that's ok). 

If you think it's right you should also update the default value on `content-search/index.js` so it's `undefined` instead of `null`

Reply:
@Antonio-Laguna do you mean something like this?

```
SearchItem.defaultProps = {
	...
	renderType: ( suggestion ) => {
		return suggestion.type === 'post_tag' ? 'tag' : suggestion.type;
	},
	...
}
```

Agreed, this looks cleaner to me, happy to add in.

Reply:
Exactly that!

Reply:
Great stuff, will get this tidied up tomorrow

Reply:
@Antonio-Laguna update just landed